Tests/API_Integation_test/TasksControllerTest.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Moq;
```

Comment:
Godt at se du fik Mock DB'en til at virke! Du må lige fortælle mig når du får tid om det var nemt nok med moq og microsoft,aspnetcore i stedet for at mocke en sqlite som jeg gjorde med deres library